bug-triage/triage-workflow.md

```diff
@@ -1,6 +1,14 @@
 # Bug Triage Overview
```

Comment:
Would also add a small bit near the start or end about asking questions - both for consulting on an issue and generally about the process/docs (which could include suggestions)